ChegeBryan/black-bandana
# #163261740 User model

#### What does this PR do?

Create a user class.

#### Description of Task to be completed?

- Write tests for the user class
- Create the user class

#### How should this be manually tested?
After cloning the repo to your local machine - check [README.md](README.md) for instructions. Check out to `ft-register-user-api-163261740` branch Run the tests using `pytest --cov=app/tests`

#### What are the relevant pivotal tracker stories?
[#163261740](https://www.pivotaltracker.com/story/show/#163261740)
